File: src/test/java/no/utgdev/diy/ex3/Ex3Config.java
package no.utgdev.diy.ex3;

import no.utgdev.diy.annotations.Bean;
import no.utgdev.diy.annotations.Import;

@Import(Ex3Config2.class)
public class Ex3Config {
    @Bean
    public Controller controller() {
        return new Controller();
    }

    @Bean
    public HelloDao helloDao() {
        return new HelloDao();
    }
}

File: src/test/kotlin/no/utgdev/diy/ex3/Ex3Config.kt
package no.utgdev.diy.ex3

//REMOVE import no.utgdev.diy.annotations.Bean;
//REMOVE import no.utgdev.diy.annotations.Import;

//REMOVE @Import([Ex3Config2::class])
class Ex3Config {
    //REMOVE @Bean
    fun controller(): Controller {
        return Controller()
    }

    //REMOVE @Bean
    fun helloDao(): HelloDao {
        return HelloDao()
    }
}